apps/client/src/atoms/EditButton.tsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+import { EditButtonStyle } from "./EditButtonStyle";
+
+const EditButton: React.FC<{ onClick: Function }> = ({ onClick }) => (
+  <EditButtonStyle onClick={() => onClick()} variant="textButton">
```

Comment:
Why not `onClick={onClick}`?
Or even `{...{onClick}}`?

Reply:
Done, see changes